lib/fs.js

```diff
@@ -1921,79 +1921,6 @@ WriteStream.prototype.close = ReadStream.prototype.close;
 // There is no shutdown() for files.
 WriteStream.prototype.destroySoon = WriteStream.prototype.end;
 
-
-// SyncWriteStream is internal. DO NOT USE.
-// Temporary hack for process.stdout and process.stderr when piped to files.
-function SyncWriteStream(fd, options) {
-  Stream.call(this);
-
-  options = options || {};
-
-  this.fd = fd;
-  this.writable = true;
-  this.readable = false;
-  this.autoClose = options.autoClose === undefined ? true : options.autoClose;
-}
-
-util.inherits(SyncWriteStream, Stream);
-
-
-// Export
-Object.defineProperty(fs, 'SyncWriteStream', {
-  configurable: true,
-  writable: true,
-  value: SyncWriteStream
-});
```

Comment:
Since there is at least some ecosystem use I'll throw a hard deprecation
warning into fs. Thanks for the review @ChAlKeR
On May 14, 2016 1:39 PM, "Сковорода Никита Андреевич" <
notifications@github.com> wrote:

> In lib/fs.js
> <https://github.com/nodejs/node/pull/6749#discussion_r63281703>:
>
> > -
> > -  this.fd = fd;
> > -  this.writable = true;
> > -  this.readable = false;
> > -  this.autoClose = options.autoClose === undefined ? true : options.autoClose;
> > -}
> > -
> > -util.inherits(SyncWriteStream, Stream);
> > -
> > -
> > -// Export
> > -Object.defineProperty(fs, 'SyncWriteStream', {
> > -  configurable: true,
> > -  writable: true,
> > -  value: SyncWriteStream
> > -});
>
> @Fishrock123 <https://github.com/Fishrock123> Done:
> https://gist.github.com/ChALkeR/f2cb74429769e91752d4b7f41b70dda2
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6749/files/1173d733a455187d47f4e4eb61a815138e2abfe0#r63281703>
>